### PR TITLE
[MM-33498] - Correct Admin Console Reference

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Changes "Admin Console" to "System Console" in the translations

#### Ticket Link
[MM-33498](https://mattermost.atlassian.net/browse/MM-33498?atlOrigin=eyJpIjoiMjJiMjgwZDI4MzEzNGFlZjhiMTEyNjM3OTJmNmU1NmIiLCJwIjoiaiJ9)

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: i18n/en.json [hunk 1]
```diff
@@ -6260,7 +6260,7 @@
   },
   {
     "id": "ent.cloud.subscription.error",
-    "translation": "Error getting cloud subscription"
+    "translation": ""
```
Comment (pablovelezvidal) [inline i18n/en.json:6263]: is this removal intended?

Comment (AGMETEOR) [reply i18n/en.json]: @pablovelezvidal fixed it

**Review by pablovelezvidal — APPROVED** [review]
